server/routes/authorization.js

```diff
@@ -37,6 +38,7 @@ const {
 const { authorize } = require('../middleware/authorize');
 
 router.post('/demo', setDemoUser, sendUser);
+router.get('/demo', setDemoUser, demoRedirect);
```

Comment:
do we need the second controller for the same path?

Reply:
The path is the same but the request method has changed. It has to be `GET` instead of `POST` because we want to have access to demo from landing page via a link not via fetch from the app.

The controller is the same `setDemoUser` because it has to do the same thing (with small additional check) only the final middleware is different. It has to be different because when the user launches demo from landing he is redirected from landing to the `app.eoc.sanddev.com/auth/demo` so at the end he has to be redirected to '/' - app dashboard. 

When a user launches a demo from 'AuthBox' he is already on the app page so he gets only user data.